src/2_RTB/VerbaleInterno_071123/content.typ

```diff
@@ -0,0 +1,85 @@
+#import "meta.typ": inizio_incontro, fine_incontro, luogo_incontro
+#import "functions.typ": glossary, team
+#let participants = csv("participants.csv")
+
+= Partecipanti
+/ Inizio incontro: #inizio_incontro
+/ Fine incontro: #fine_incontro
+/ Luogo incontro: #luogo_incontro
+
+#table(
+  columns: (3fr, 1fr),
+  [*Nome*], [*Durata presenza*],
+  ..participants.flatten()
+)
+= Sintesi Elaborazione Incontro
+
+== Assegnazione dei ruoli
+Su proposta dei componenti è stata assegnata questa prima distribuzione dei ruoli:
+- Responsabile: Riccardo Toniolo;
+- Amministratore: Matteo Rango;
+- Analista: Simone Caregnato;
+- Analista: Nancy Kalaj;
+- Verificatore: Riccardo Constantin;
+- Verificatore: Giacomo D'Ovidio;
+Per il momento i ruoli di Progettista e Programmatore non sono stati assegnati, in quanto non sono significativi per questa fase di progetto.
+
+== Verbali degli Stand-Up
+Dato l'importanza di registrare le decisioni prese durante gli incontri, si è sollevato il dibattito sulla necessità di verbalizzare gli Stand-Up. La decisione è stata quella di verbalizzare solo gli incontri in cui sono presenti quattro o più partecipanti, che quindi coinvolgano la maggior parte del gruppo di lavoro; questo perché si tratterà di incontri più importanti, in cui si prenderanno decisioni più rilevanti. Questa prassi dovrà essere aggiunta alle #glossary("NdP") e il compito è stato assegnato a Matteo Rango.
+
+#pagebreak()
+
+== Separazione delle Repository
+Per quanto riguarda questo aspetto, abbiamo optato per la suddivisione in due repository distinte: una dedicata al codice, denominata _InnovaCity_, e un'altra per la documentazione, chiamata _Docs_. Questa scelta mira a semplificare la gestione complessiva del progetto e a evitare dispersione all'interno di una singola repository.
```

Comment:
bisogna chiedere a tullio se nella docs ci vada anche quella relativa allo sviluppo software.

Reply:
Abbiamo stabilito che teniamo 2 repo distinte per documenti e per sviluppo del sw

Reply:
per quanto riguarda la documentazione del codice in che repo va a finire? credo fosse una cosa da chidere a tullio. 
Comunque per me va bene anche non specificarlo qui equindi tenere cosi il documento.